src/main/java/org/reaktivity/nukleus/kafka/internal/memory/DefaultMemoryManager.java

```diff
@@ -16,7 +16,7 @@
 package org.reaktivity.nukleus.kafka.internal.memory;
 
 import static java.lang.Integer.highestOneBit;
-import static java.lang.Integer.numberOfTrailingZeros;
+import static java.lang.Long.numberOfTrailingZeros;
```

Comment:
Also need to change import to `Long.highestOneBit` ?

Reply:
Correct, done